src/select-menu/docs/examples/SelectMenu-multi.example

```diff
@@ -0,0 +1,46 @@
+<Manager>
+       {({ setState, state }) => (
+              <SelectMenu
+                title="Select name"
+                options={options}
+                selected={state.selected}
+                onSelect={item => {
+            let selected
+            if (Array.isArray(state.selected)) {
+              selected = [...state.selected, item.value]
+            } else {
+              selected = arrify(item.value)
+            }
+            setState({
+              selected,
+              selectedNames: selected.toString().substr(0, 25) + '...'
+            })
+          }}
+          onDeselect={item => {
+            const DeselectedItemIndex = state.selected.indexOf(item.value)
```

Comment:
`DeselectedItemIndex` => `deselectedItemIndex`. UpperCamelCase is only for class names (components) and enum like objects.